meltano.yml

```diff
@@ -27,6 +35,10 @@ plugins:
           description: |
             Make the replication key optional. If set to true, the tap will
             not fail if the replication key is not found in a document.
+        - name: is_timestamp_replication_key
```

Comment:
This works if all streams use timestamp replication keys, but wouldn't if there was a mix. Can this be detected automatically from the column type, or configured in stream metadata?